src/plugins/query_enhancements/public/query_assist/utils/create_extension.tsx

```diff
@@ -93,6 +93,11 @@ const getAvailableLanguages$ = (http: HttpSetup, data: DataPublicPluginSetup) =>
     })
   );
 
+const queryAssistantSupportedDatasetTypes = [
+  DEFAULT_DATA.SET_TYPES.INDEX,
+  DEFAULT_DATA.SET_TYPES.INDEX_PATTERN,
```

Comment:
thanks for doing this but should this be apart of the switch map on line 87

Reply:
Sure, updated.